src/util/statistics.cpp

```diff
@@ -126,6 +126,7 @@ void StatisticsBase::safeFlushInformation(int fd) const {
       safe_print(fd, s_regDelim);
     }
     s->safeFlushStat(fd);
+    safe_print(fd, "\n");
   }
   safe_print(fd, "\n");
```

Comment:
Nit: we can probably remove this one now.

Reply:
I kept it to match the newline behaviour of the non-safe printing.

Reply:
I removed it now for both versions.